auto_submit/lib/model/big_query_pull_request_record.dart

```diff
@@ -0,0 +1,67 @@
+// Copyright 2022 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+class PullRequestRecord {
+  PullRequestRecord({
+    this.prCreatedTimestamp,
+    this.prLandedTimestamp,
+    this.organization,
+    this.repository,
+    this.author,
+    this.prId,
+    this.prCommit,
+    this.prRequestType,
+  });
+
+  int? prCreatedTimestamp;
+  int? prLandedTimestamp;
+  String? organization;
+  String? repository;
+  String? author;
+  int? prId;
```

Comment:
I think you mean prNumber. prId is something much more specific and unreadable

Reply:
You are right. Let me change it to number.